flotta-agent.spec

```diff
@@ -73,28 +57,27 @@ install ./bin/device-worker %{buildroot}%{_libexecdir}/yggdrasil/device-worker
 install ./bin/device-worker-race %{buildroot}%{_libexecdir}/yggdrasil/device-worker-race
 make install-worker-config USER=%{flotta_user} HOME=/home/%{flotta_user} LIBEXECDIR=%{_libexecdir} BUILDROOT=%{buildroot} SYSCONFDIR=%{_sysconfdir}
```

Comment:
I think we need to change the `/home/%{flotta_user}` here to `/var/home/%{flotta_user}`